### PR TITLE
Initial Partial Updates on t-echo

### PR DESCRIPTION
**Pull request by ixt**
#3075 

Partial Updates for t-echo, The boot screen is very faint. And we don't (to my knowledge) do a full refresh at some point to help with ghosting but this should be good for now. I can't believe it was basically working. I'll find some time to write something for full updates (maybe on manual loop of screens and after n-partial updates)

### CODE REVIEW

**Comment by markbirss** [comment]
@ixt it normal for some e-inks to have faint boot screen

yes, a counter but it must goto the tros tasks i think

File: src/graphics/EInkDisplay2.cpp [hunk 1]
```diff
@@ -55,7 +55,7 @@ GxEPD2_BW<TECHO_DISPLAY_MODEL, TECHO_DISPLAY_MODEL::HEIGHT> *adafruitDisplay;
 EInkDisplay::EInkDisplay(uint8_t address, int sda, int scl, OLEDDISPLAY_GEOMETRY geometry, HW_I2C i2cBus)
 {
 #if defined(TTGO_T_ECHO)
-    setGeometry(GEOMETRY_RAWMODE, TECHO_DISPLAY_MODEL::WIDTH, TECHO_DISPLAY_MODEL::HEIGHT);
```
Comment (thebentern) [inline src/graphics/EInkDisplay2.cpp:58]: Is this change necessary?

Comment (markbirss) [reply src/graphics/EInkDisplay2.cpp:58]: I was looking at all the e-ink support as raised by the previous ESP32-S3-PICO PR. I wanted to look at supply these geometry values in the platformio.ini variant files which would reduce the checks so it ok for now

Comment (ixt) [reply src/graphics/EInkDisplay2.cpp:58]: This was the change actually that made it all work for me (aside from the explicitly partial update parts). The why is an open question to me though
